README.md

```diff
@@ -52,3 +52,7 @@ Testing plural --> singular
 The singular of feet is foot
 The singular of computers is computer
 ```
+
+## pray Thanks
```

Comment:
```suggestion
## 🙏 Thanks
```

Reply:
If you can apply the suggestion above I'll merge, and again, thanks a lot for your contribution @vixalien